src/components/Ref/Ref.tsx

```diff
@@ -2,11 +2,19 @@ import * as PropTypes from 'prop-types'
 import * as React from 'react'
 import { findDOMNode } from 'react-dom'
 
-import { ReactChildren } from '../../../types/utils'
 import { handleRef } from '../../lib'
 
 export interface RefProps {
-  children?: ReactChildren
+  /**
+   *  Used to set content when using childrenApi - internal only
+   *  @docSiteIgnore
+   */
+  children?: React.ReactChild
```

Comment:
I would not this see to be internal only, as _in case if Ref will be exposed to the client from Stardust_, either of the following is absolutely legal:

```jsx
<Ref innerRef={...}>
  <MyCustomComponent>
</Ref>
```

or its equivalent:

```jsx
<Ref innerRef={...} children={<MyCustomComponent>} />
```

So, thus, would suggest to use already existing prop type here as well (`ChildrenComponentProps`), as its description for `children` will be an exact match for this component as well